pyomo/core/tests/unit/test_set.py

```diff
@@ -334,12 +334,12 @@ def test_setdefault(self):
     def test_indices(self):
```

Comment:
It looks like `black` might have gone a bit crazy on this file. Can I please ask you to revert it and add back the lines you actually changed?